### ISSUE TITLE
make more use of Cassandane::Mboxname

### ISSUE THREAD

**Issue by elliefm** [cyrusimap/cassandane#61]
seems like it's probably useful for generalising a lot of the altnamespace/unixhierarchysep/etc munging

**Comment by brong** [comment]
Yes, this is one of the "make the codebase more flexible" issues we should work on.
